hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -73,19 +73,23 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
 
 public:
 
-  static QString userEmail() { return "ServiceHootApiDbReaderTest@hoottestcpp.org"; }
+  QString userEmail() { return QString("%1.ServiceHootApiDbReaderTest@hoottestcpp.org").arg(testName); }
```

Comment:
Since a lot of these tests create and then remove the user along with the map, the unique test name is added to the email to make the user's email (and further down the user's name) unique per test.